src/main/java/com/kodilla/ecommercee/domain/UserDto.java

```diff
@@ -7,25 +7,27 @@
 import java.util.List;
 
 @Getter
-@AllArgsConstructor
+//@AllArgsConstructor
 public class UserDto {
     private Long id;
     private String username;
     private int status;
     private String userKey;
     private LocalDateTime expirationTime;
-    private List<Order> listOfOrders;
+    //private List<Order> listOfOrders;
 
-    public void setListOfOrders(List<Order> listOfOrders) {
-        this.listOfOrders = listOfOrders;
-    }
+    //public void setListOfOrders(List<Order> listOfOrders) {
```

Comment:
Tylko może powyrzucaj takie zakomentowane rzeczy, bo się bałagan robi

Reply:
Prawda, zafiksowalem się na tych encjach i relacjach, a to szło w którymś pierwszym commicie ;)